### PR TITLE
Face blur using pose

### PR DESCRIPTION
**Pull request by gangooteli**
# Description
Added code to perform face blur using pose coordinates

User story # (US link)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [ ] Test A

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests
- [ ] New and existing unit tests pass locally with my changes
- [ ] My Code is review by 1 People

### CODE REVIEW

File: cgmml/models/HRNET/inference-config-hrnet_w32.yaml [hunk 1]
```diff
@@ -69,8 +69,8 @@ MODEL:
 
 TEST:
   POST_PROCESS: true
-  POSE_DRAW : false
-  DATA_PATH: 'data/path/version_wise_jpg'
+  POSE_DRAW : true
+  DATA_PATH: '/home/nik/Projects/cgm-all/cgm-ml/data/path/version_wise_jpg/'
```
Comment (prajurock) [inline cgmml/models/HRNET/inference-config-hrnet_w32.yaml:73]: Contains your local path.
Please change

File: cgmml/models/HRNET/code/utils/demo_utils.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    img = cv2.imread(img_path)
+    # Convert it to greyscale
+    img_grey = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+    # Threshold the image
+    ret, thresh = cv2.threshold(img_grey, 50, 255, 0)
+    # Find the contours
+    contours, hierarchy = cv2.findContours(thresh, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+    # For each contour, find the convex hull and draw it
+    # on the original image.
+    print("len of contours ", len(contours))
```
Comment (prajurock) [inline cgmml/models/HRNET/code/utils/demo_utils.py:36]: Why we are printing at some where and logging at some place.

**Review by prajurock — CHANGES_REQUESTED** [review]

File: cgmml/models/HRNET/code/utils/demo_utils.py (hunk 2 above)
Comment (gangooteli) [reply cgmml/models/HRNET/code/utils/demo_utils.py]: Fixed

File: cgmml/models/HRNET/inference-config-hrnet_w32.yaml (hunk 1 above)
Comment (gangooteli) [reply cgmml/models/HRNET/inference-config-hrnet_w32.yaml]: Fixed

**Review by prajurock — APPROVED** [review]

**Pull request merged**